prov/gni/src/gnix_vc.c

```diff
@@ -281,34 +281,34 @@ static void __gnix_vc_pack_conn_req(char *sbuf,
 	      sizeof(gni_mem_handle_t);
 	assert(len <= GNIX_CM_NIC_MAX_MSG_SIZE);
 
-	memcpy(cptr, &rtype, sizeof(rtype));
+	*((uint8_t *)cptr) = rtype;
 	cptr += sizeof(rtype);
```

Comment:
In general, I'd like to see more samples of this analysis. If there was only one measurement (before and after), I'd like to see more measurements. What was the difference in the maximum, minimum and average times between multiple runs? How can you prove that the difference in time isn't attributed to network noise?